src/test/config.sol

```diff
@@ -1371,11 +1371,11 @@ contract Config {
             lerp:         false
         });
         afterSpell.collaterals["RWA007-A"] = CollateralValues({
-            aL_enabled:   false,
-            aL_line:      0,
-            aL_gap:       0,
-            aL_ttl:       0,
-            line:         1_000_000,
+            aL_enabled:   true,
+            aL_line:      250 * MILLION,
+            aL_gap:       50 * MILLION,
+            aL_ttl:       604_800, // 1 week
+            line:         0,
```

Comment:
Point on this in my checklist does setIlk() on the IAM change this?

Reply:
I think it's more that the test no longer validates line once autoline is on (since autoline adjusts this).

Reply:
You're correct, this is an internal bool used by DssSpell.t.base to determine what sets of config values to check.

Either way IMO I would leave line as 1M since that is the actual value unless we are changing this - is there a previous pattern for this? 

```suggestion
            line:         1_000_000,
```